oauth2client/service_account.py

```diff
@@ -461,3 +465,96 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+def _datetime_to_secs(utc_time):
+   # time_delta.total_seconds() not supported in Python 2.6
```

Comment:
I am big time :-1: on `TODO` comments in code. There is a world-class issue tracker right here, put the `TODO` there (and you can even link to the PR / line of code with commit hash and all).